src/components/Explore/ShareButton.tsx

```diff
@@ -0,0 +1,139 @@
+import { useOnClickOutside } from 'hooks/useOnClickOutside'
+import { darken } from 'polished'
+import { useRef, useState } from 'react'
+import { Check, Link, Share, Twitter } from 'react-feather'
+import { useModalIsOpen, useToggleModal } from 'state/application/hooks'
+import { ApplicationModal } from 'state/application/reducer'
+import styled, { useTheme } from 'styled-components/macro'
+
+const TWITTER_WIDTH = 560
+const TWITTER_HEIGHT = 480
+const SHADOW = '0px 1px 6px rgba(0, 0, 0, 0.9), 0px 8px 12px rgba(13, 14, 14, 0.8)'
```

Comment:
I thiiiink we have a standard box shadow somewhere

Reply:
looks like currently a lot of files use hard-coded, but as part of the next thing I'm working on (theme update), we have a standardized box shadow. So I will update this when that PR pushes!